lib/lolcommits/plugins/lol_flowdock.rb

```diff
@@ -0,0 +1,70 @@
+# -*- encoding : utf-8 -*-
+require 'rest_client'
+
+module Lolcommits
+  class LolFlowdock < Plugin
+    ENDPOINT_URL = 'api.flowdock.com/flows/'.freeze
+    RETRY_COUNT  = 2
+
+    def self.name
+      'flowdock'
+    end
+
+    def self.runner_order
+      :postcapture
+    end
+
+    def configured?
+      !configuration['access_token'].nil?
+    end
+
+    def configure
+      print "Open the URL below and issue a token for your user (Personal API token):\n"
+      print "https://flowdock.com/account/tokens\n"
+      print "Enter the generated token below, then press enter: \n"
+      code = STDIN.gets.to_s.strip
+      print "Enter the machine name of the flow you want to post to from this repo.\n"
+      print "Go to https://www.flowdock.com/account and click Flows, then click the flow, then get the machine name from the URL:\n"
+      flow = STDIN.gets.to_s.strip
+      print "Enter the name of the organization for this Flowdock account.\n"
+      organization = STDIN.gets.to_s.strip
+
```

Comment:
Along with strip, I think you also need to `downcase` the flow and organisation captured above.  

When I tried with my test new flowdock account, I mistakenly configured with capitalised letters and the endpoint returned 403 (so it is case sensitive).